lib/spack/docs/module_file_support.rst

```diff
@@ -641,11 +658,6 @@ The allowed values for the ``autoload`` statement are either ``none``,
 
   .. code-block:: yaml
```

Comment:
This should be moved up to the other code block

Reply:
Fixed